modules/cas_cache/exp_obj.c

```diff
@@ -31,7 +31,7 @@ static inline void bd_release_from_disk(struct block_device *bdev,
 	return bd_unlink_disk_holder(bdev, disk);
 }
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 18, 0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 18, 0) || RHEL_MAJOR >= 9
```

Comment:
@MJAsadi72 could you please test whether the following code works on Rockylinux 9? I've tested it on a non-RHEL kernel and it works fine. 

```suggestion
#ifdef RHEL_MAJOR
       #if RHEL_MAJOR >= 9
               #define KRETURN(x)      return
               #define MAKE_RQ_RET_TYPE void
       #endif
 #else
       #if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 18, 0)
               #define KRETURN(x)      return
               #define MAKE_RQ_RET_TYPE void
       #elif LINUX_VERSION_CODE >= KERNEL_VERSION(4, 3, 0)
               #define KRETURN(x)      ({ return (x); })
               #define MAKE_RQ_RET_TYPE blk_qc_t
       #elif LINUX_VERSION_CODE >= KERNEL_VERSION(3, 2, 0)
               #define KRETURN(x)      return
               #define MAKE_RQ_RET_TYPE void
       #else
               #define KRETURN(x)      ({ return (x); })
               #define MAKE_RQ_RET_TYPE int
       #endif
#endif
```